.lando.yml

```diff
@@ -0,0 +1,45 @@
+name: drupal-orange-project
+recipe: drupal8
+config:
+  webroot: web
+  xdebug: true
+
```

Comment:
Lando uses apache by default with the drupal8 recipe, right? I'd suggest adding `via: nginx` to this `config` section to use nginx instead since that's what we use everywhere.